__tests__/main.test.ts

```diff
@@ -1,24 +1,10 @@
 import * as cp from "child_process";
 import * as path from "path";
 import * as process from "process";
-import { expect, test } from "@jest/globals";
-import { wait } from "../src/wait";
-
-test("throws invalid number", async () => {
-  const input = parseInt("foo", 10);
-  await expect(wait(input)).rejects.toThrow("milliseconds not a number");
-});
-
-test("wait 500 ms", async () => {
-  const start = new Date();
-  await wait(500);
-  const end = new Date();
-  const delta = Math.abs(end.getTime() - start.getTime());
-  expect(delta).toBeGreaterThan(450);
-});
+import { test } from "@jest/globals";
 
 // shows how the runner will run a javascript action with env / stdout protocol
-test("test runs", () => {
+test.skip("test runs", () => {
```

Comment:
This test runs the action locally and ensures it exits cleanly. I think a better version of this is an integration test, similar to https://github.com/guardian/actions-assume-aws-role/pull/6/files. Skipping for now.